### PR TITLE
Plugin Upgrade Instructions in  Rolling Upgrade Doc

### PR DESCRIPTION
**Pull request by tylerfontaine**
Added plugin upgrade instructions per #15389

### CODE REVIEW

File: docs/reference/setup/rolling_upgrade.asciidoc [hunk 1]
```diff
@@ -87,7 +87,26 @@ To upgrade using a zip or compressed tarball:
     to your new installation, or configure the location of the data directory
     in the `config/elasticsearch.yml` file, with the `path.data` setting.
 
-==== Step 4: Start the upgraded node
+==== Step 4: Upgrade Plugins
+
+Now that the elasticsearch packages have been updated, you should update any plugins you have installed as well. Please make sure you check the release notes for any plugins before installing them. To update the plugins:
+
+*   First, remove the exising plugin with `bin/plugin remove <plugin name>`.
```
Comment (gmoskovicz) [inline docs/reference/setup/rolling_upgrade.asciidoc]: I will suggest to first get all the installed plugins:

```
➜  elasticsearch-2.1.0  bin/plugin list
Installed plugins in /Users/Gabriel/Documents/ElasticSearch/elasticsearch-2.1.0/plugins:
    - license
    - marvel-agent
    - shield
```

Then, for each plugin listed in the list output, do all this steps.

File: docs/reference/setup/rolling_upgrade.asciidoc [hunk 2]
```diff
@@ -87,7 +87,26 @@ To upgrade using a zip or compressed tarball:
...
+
+*   Next, install the updated plugin with `bin/plugin install <plugin name>`.
```
Comment (gmoskovicz) [inline docs/reference/setup/rolling_upgrade.asciidoc]: Be careful, some plugins (community) needs a full URI and/or branch/version. Example KOPF

**Comment by dakrone** [comment]
@tylerfontaine I think this needs to be rebased on master to get rid of the conflicts, additionally, `bin/plugin` is now `bin/elasticsearch-plugin` I believe

**Comment by clintongormley** [comment]
I added these steps in https://github.com/elastic/elasticsearch/commit/06604708d4bfc56eb397c6003567886bb1ce0114

**Pull request closed without merging**